docs/variables/sender.md

```diff
@@ -4,118 +4,220 @@ id: sender
 
 # $(sender)
 
-The `$(sender)` variable represents the user who sent the message.
+Returns a range of metadata around the sender's channel that can be accessed via multiple different variable members for each unique piece of information.
```

Comment:
i think this should also cover what just `$(sender)` does, which is equiv to `$(sender.display_name)`

Reply:
I'll add a new list item for `$(sender)`.